spec/features/visitors/password_recovery_spec.rb

```diff
@@ -40,11 +40,8 @@ def reset_password_and_sign_back_in(user, password = 'a really long password')
       user = create(:user, :unconfirmed)
       confirm_last_user
       reset_email
-      visit new_user_password_path
-      fill_in 'Email', with: user.email
-      click_button t('forms.buttons.continue')
-      open_last_email
-      click_email_link_matching(/reset_password_token/)
+
+      trigger_reset_password_and_click_email_link(user.email)
```

Comment:
YES! Also, noticing that we define this method at the bottom of the file. I am used to that convention but noticed we were defining a helper method at the top of this file. Thoughts on which we should do, generally? Happy with either as long as we're consistent